### ISSUE TITLE
running

### ISSUE THREAD

**Issue by unintends** [cop-discord/pretend-src#4]
any guide on running this for testing purposes?

**Comment by cop-discord** [comment]
no there aren't any as running this is not something ur supposed to do, this bot source is dog shit and I wouldn't even run it on a vm tbh

**Comment by darkistifyy** [comment]
> any guide on running this for testing purposes?

no, and anyway you shouldnt be running this shitty src in the place. i wouldnt put it past me to blow up my fucking pc icl
